### PR TITLE
Add usernameisvince's pin to the map

### PR DESCRIPTION
**Pull request by usernameisvince**
Adding SD to the map!

### CODE REVIEW

File: _pins/usernameisvince.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+---
+githubHandle: usernameisvince
+latitude: 32.715738
+longitude: -117.161084
```
Comment (brianamarie) [inline _pins/usernameisvince.json]: Hi @usernameisvince! Try adding a line at the bottom with the three `---`. Let me know if you want or need any help. 😄

Comment (usernameisvince) [inline _pins/usernameisvince.json]: I'm new and would like some help please.

**Comment by brianamarie** [comment]
@usernameisvince Looks like you don't need any help, you've got it handled! Nice work 😄 👍

**Comment by usernameisvince** [comment]
Success!

**Pull request merged**